content/blog/2022/12/kubewarden-2022-wrapped.md

```diff
@@ -0,0 +1,119 @@
+[CLOMonitor](https://clomonitor.io/projects/cncf/kubewarden).
+Thanks to CLOMonitor, Kubewarden's users have a clear picture of the project health
+and can assess its development and security practices.
+
+## Pod Security Policy replacement
+
+During the month of August, Kubernetes 1.25 release finally removed
+[Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/).
+
+This event was planned and advertised for a long time. Due to this,
+the Kubewarden developers were able to provide
+[tooling, policies, and documentation](https://docs.kubewarden.io/tasksDir/psp-migration)
+to smoothen the migration process from Kubernetes Pod Security Policies to
+Kubewarden policies.
+
+## Secure supply chain
+
+The topic of secure supply chain started to emerge near the end of 2021, but it
+was during this year that many projects and organizations tackled it.
+
+The [sigstore](https://sigstore.dev) project proved to be one of the pillars of
+these initiatives.
+Kubewarden invested early in the Sigstore project, with some of our maintainers being
+active also within the Sigstore community.
+
+During the last year Kubewarden embraced Sigstore to
+[securely distribute its own assets](/blog/2022/04/securing-kubewarden-policies/),
+like container images, policies, helm charts and cli utilities.
+Moreover, we exposed Sigstore capabilities to our policy authors and provided
+a [ready to use policy](https://artifacthub.io/packages/kubewarden/verify-image-signatures/verify-image-signatures)
+that can be used to vet the container images used by Kubernetes workloads.
+
+Sigstore provides different ways to sign artifacts: with
+traditional keys,
+[keyless mode](/blog/2022/05/verifying-image-signatures/),
+[keyless mode from a GitHub Action workflow](/blog/2022/07/verify-signatures-with-gha-and-prefix/) and
+[self-signed certificates](/blog/2022/12/sigstore-certificate-verification/)
+stored on a physical token or provided by a KMS.
+During the last year we kept extending Kubewarden to ensure all these scenarios would be
+available also to our users.
+
+## Other notable features
+
+A stream of improvements have been made to the Kubewarden project during the last
+year.
+
+The Kubernetes integration was further expanded by introducing the new
+[`AdmissionPolicy`](/blog/2022/03/admission-policy/) Custom Resource. This allows
+regular Kubernetes users to define policies that are scoped to a specific
+Namespace.
+
+Next, we introduced a new operating mode for all our policies: the
+["monitor"](/blog/2022/05/monitor-mode/) mode. This allows Kubernetes administrators
+to better understand how a policy would impact their existing clusters once being
+enforced.
+
+More recently, the Kubewarden developers simplified the process of deploying
+Kubewarden inside of [air gapped](/blog/2022/11/airgap/) environments.
+
+## New policies
+
+During the last months we created many ready to use policies.
+My favorite ones are:
```

Comment:
Since this is a project blog, I think we should refrain from the usage of first-person pronouns like My/I and should refer to the collective. This should, normally, be the case unless we're publishing a case study by a user.
```suggestion
Some of them are:
```